src/__test__/pages/experiments/[experimentId]/data-processing/index.test.jsx

```diff
@@ -143,14 +156,14 @@ describe('DataProcessingPage', () => {
     const card = page.find('Card');
     expect(card.length).toEqual(1);
 
-    const runFilterButton = page.find('#runFilterButton').filter('Button');
-    expect(runFilterButton.length).toEqual(1);
+    const runDataProcessingButton = page.find('#runDataProcessingButton').filter('Button');
+    expect(runDataProcessingButton.length).toEqual(1);
 
     // Run filter is disabled initially
```

Comment:
[boulder] if I understood correctly, this is now testing if the button is disabled when the pipeline status is 'succeeded' so the comment should be updated / removed

Reply:
updated to be more accurate